src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -776,6 +787,89 @@ public void testHaltOnExceptionOff() throws Exception {
         verify(checker, filePath, filePath, expected);
     }
 
+    @Test
+    public void testCheckerProcessCallAllNeededMethodsOfChecks() throws Exception {
```

Comment:
@Nimfadora Since this item isn't being shown as `outdated`, it isn't done. :)

Hint:
> testCheckerProcessCallAllNeededMethodsOf**Checks**